src/ert/run_models/base_run_model.py

```diff
@@ -303,33 +301,20 @@ def isIndeterminate(self) -> bool:
     def checkHaveSufficientRealizations(self, num_successful_realizations: int) -> None:
         if num_successful_realizations == 0:
             raise ErtRunError("Experiment failed! All realizations failed!")
-        if (
-            not self.ert()
-            .analysisConfig()
-            .have_enough_realisations(num_successful_realizations)
+        if not self.ert.analysisConfig().have_enough_realisations(
```

Comment:
Looks like we can get limit the use of `EnkfMain` even more and use the facade here:
```suggestion
        if not self.facade.get_analysis_config().have_enough_realisations(
```

Reply:
Is it better to use the facade?